modules/ROOT/pages/admin-center.adoc

```diff
@@ -128,14 +128,33 @@ You can add more resource metrics to the **Monitor** view by selecting the **Edi
 [#batch]
 === Manage Java batch jobs with the Java Batch tool
 
-If you configure the feature:batchManagement[display=Batch Management] feature, you can access the **Java Batch** tool. With this tool, you can view the progress and status of your Java batch jobs, manage their instances, and view their log files. The following screen capture shows the **Java Batch** tool:
+If you configure�the feature:batchManagement[display=Batch Management] feature, you can access the **Java Batch** tool. With this tool, you can view the progress and status of your Java batch jobs, manage their instances, and view their log files. The following screen capture shows the **Java Batch** tool:
 
 image::ui_javaBatchTool.png[align="center"]
 
 Each batch job has an **Actions** icon, which you can select to stop, restart, or purge the job, and a **View Logs** icon, which you can select to view the job logs. To view details and metrics for a batch job, hover over the **Batch Job Name** and select an option from the tooltip menu.
 
 If batch jobs or job logs are on remote servers, link:/guides/cors.html[configure cross origin region sharing (CORS)] on each remote server. CORS enables Admin Center to request job information from remote servers.
 
+Note too that the batch features use a custom authorization specific to batch and role membership must be granted separately from the roles needed to access the Admin Center in the first place.  
+
+E.g. a 'batchadmin' user configured like:
+
+[source,xml]
+----
+  <reader-role>
+      <user>batchadmin</user>
+  </reader-role>
+
+  <authorization-roles id="com.ibm.ws.batch">
+      <security-role name="batchAdmin">
+          <user name="batchadmin"/>
+      </security-role>
+  </authorization-roles>
+----
```

Comment:
also want to clarify- even a user in the reader role can edit batch jobs if they have the proper `com.ibm.ws.batch` auth? I thought that reader role gave you read-only access to the Admin center tools- is that not the case with Java Batch?

Reply:
Yeah, this was exactly the nuance I was trying to illustrate.    I think it makes sense to basically defer to batch's custom auth here but let me also tag @mbroz2  to clarify if this is truly the intended design.

Reply:
reader-role is not an Admin Center concept, but rather a Liberty one (aka, it's applied to all (well almost all) APIs, including Admin Center ones, unless the API uses their own set of authorization, like batch).  
I would expect a user to be be able to have the 'reader' role and the 'batchAdmin' role, allowing them to view all the tools in admin center, but also perform batch admin job operations in the java batch tool.... but that's only true if no other/non-java-batch APIs are called that would do anything other then a GET operation.  I'm guessing there aren't, but @kinueng or @k8vance88 could confirm.  One way to test this would be to just run the Admin Center Java Batch FAT tests, and change the user's role from (what I'm guessing is) admin to reader.

Of course the cheapest route is to simply change the example to asign the batchadmin user to the admin role instead of the reader role :)

Reply:
Yeah, this was exactly the nuance I was trying to illustrate. I think it makes sense to basically defer to batch's custom auth here but let me also tag @mbroz2 to clarify if this is truly the intended design.

Reply:
Thx @mbroz2.   Yeah, if we just want the sample snippet to help a single developer copy/paste working config we probably just want to use the admin role.   I don't think you need 'admin' for anything purely batch related but it can be useful e.g. for  **ibm/api/validation**.

I was trying to capture the angle "what is the minimum auth required?" for users trying to use the roles in a fine-grained manner.   

Maybe a good compromise would be to show the sample snippet with admin role but mention in a sentence that you can use reader-role for batch operations.